packages/compiler-cli/src/ngtsc/typecheck/src/oob.ts

```diff
@@ -340,6 +348,34 @@ export class OutOfBandDiagnosticRecorderImpl implements OutOfBandDiagnosticRecor
         ts.DiagnosticCategory.Error, ngErrorCode(ErrorCode.INACCESSIBLE_DEFERRED_TRIGGER_ELEMENT),
         message));
   }
+
+  controlFlowPreventingContentProjection(
+      templateId: TemplateId, projectionNode: TmplAstElement|TmplAstTemplate, componentName: string,
+      slotSelector: string, controlFlowNode: TmplAstIfBlockBranch|TmplAstForLoopBlock,
+      preservesWhitespaces: boolean): void {
+    const blockName = controlFlowNode instanceof TmplAstIfBlockBranch ? '@if' : '@for';
+    const lines = [
+      `Node matches the "${slotSelector}" slot of the "${
+          componentName}" component, but will not be projected into the specifc slot because the surrounding ${
```

Comment:
```suggestion
          componentName}" component, but will not be projected into the specific slot because the surrounding ${
```

I approved but missed this update, small typo here